sunpy/visualization/tests/test_drawing.py

```diff
@@ -18,6 +18,21 @@ def aia171_test_map():
     return Map(get_test_filepath('aia_171_level1.fits'))
 
 
+@pytest.fixture
+def cropped_aia193_sample_map():
+    import sunpy.data.sample
+    m = sunpy.map.Map(sunpy.data.sample.AIA_193_JUN2012)
```

Comment:
What is this sample data file?!

Reply:
https://docs.sunpy.org/en/stable/generated/gallery/map_transformations/reprojection_aia_euvi_mosaic.html#sphx-glr-generated-gallery-map-transformations-reprojection-aia-euvi-mosaic-py